src/main/java/today/sesac/shoutify/post/entity/Post.java

```diff
@@ -0,0 +1,88 @@
+package today.sesac.shoutify.post.entity;
+
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import today.sesac.shoutify.global.domain.BaseEntity;
+
+/**
+ * 게시판의 게시글을 관리하는 도메인입니다.
+ */
+@Entity //엔티티임을 표시
+@Table(name = "posts") //엔티티<->매핑할 테이블 , 이름 생략시 엔티티 이름 사용, 유니크 제약조건
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED) //매개변수 없는 생성자를 protected로 만들어줌
+
+public class Post extends BaseEntity {
+
+    @Id //기본키
+    @GeneratedValue(strategy = GenerationType.IDENTITY) //auto_increment
+    private Long id;
+    /**
+     * 비속어 ai 수정 전 게시물 내용입니다.
+     */
+    @Column(nullable = false, columnDefinition = "TEXT")
+    private String afterContents;
+    /**
+     * 게시물을 작성한 회원입니다
+     */
+//    @ManyToOne
+//    @JoinColumn(name = "author", nullable = false)
+//    private User user;
+    /**
+     * 게시물의 제목입니다.
+     */
+    @Column(length = 255)
+    private String title;
```

Comment:
P3
이것도 명시적인 값을 좋아하시나 해서 말씀드리진 않았는데, 혹시 모르실까 해 남깁니다.
jpa에서 String으로 선언 시 기본적으로 VARCHAR(255)로 선언돼서 255면 따로 남겨주지 않아도 됩니다.
```suggestion
    private String title;
```

Reply:
오호 몰랐네요,, 감사합니다!